### PR TITLE
djvulibre: add livecheckable

### PR DESCRIPTION
**Pull request by chenrui333**
# before

```
$ brew livecheck djvulibre
djvulibre (guessed) : 3.5.27 ==> 18n-mergepoint
```

<details>
<summary>brew livecheck djvulibre --debug</summary>

```
Trying with url https://git.code.sf.net/p/djvu/djvulibre-git.git
Possible git repo detected at https://git.code.sf.net/p/djvu/djvulibre-git.git
cygwin_version_3_5_17 => #<Version:0x00007fb88d188060 @version="3_5_17", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 17>]>
i18n-branch-point => #<Version:0x00007fb88d193208 @version="18n-branch-point", @tokens=[#<Version::NumericToken 18>, #<Version::StringToken "n">, #<Version::StringToken "branch">, #<Version::PatchToken "p">, #<Version::StringToken "oint">]>
i18n-mergepoint => #<Version:0x00007fb88d193140 @version="18n-mergepoint", @tokens=[#<Version::NumericToken 18>, #<Version::StringToken "n">, #<Version::StringToken "mergepoint">]>
release.3.5.1 => #<Version:0x00007fb88d192ee8 @version="3.5.1", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 1>]>
release.3.5.10 => #<Version:0x00007fb88d192e20 @version="3.5.10", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 10>]>
release.3.5.11 => #<Version:0x00007fb88d192d58 @version="3.5.11", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 11>]>
release.3.5.12 => #<Version:0x00007fb88d192c90 @version="3.5.12", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 12>]>
release.3.5.13 => #<Version:0x00007fb88d192b78 @version="3.5.13", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 13>]>
release.3.5.13.pre14 => #<Version:0x00007fb88d192ab0 @version="3.5.13.pre14", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 13>, #<Version::PreToken "pre14">]>
release.3.5.14 => #<Version:0x00007fb88d1929e8 @version="3.5.14", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 14>]>
release.3.5.15 => #<Version:0x00007fb88d192920 @version="3.5.15", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 15>]>
release.3.5.16 => #<Version:0x00007fb88d192858 @version="3.5.16", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 16>]>
release.3.5.17 => #<Version:0x00007fb88d192790 @version="3.5.17", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 17>]>
release.3.5.18 => #<Version:0x00007fb88d1926a0 @version="3.5.18", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 18>]>
release.3.5.19 => #<Version:0x00007fb88d1925d8 @version="3.5.19", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 19>]>
release.3.5.2 => #<Version:0x00007fb88d1924e8 @version="3.5.2", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 2>]>
release.3.5.20 => #<Version:0x00007fb88d1923d0 @version="3.5.20", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 20>]>
release.3.5.20.2 => #<Version:0x00007fb88d1922b8 @version="3.5.20.2", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 20>, #<Version::NumericToken 2>]>
release.3.5.20.4 => #<Version:0x00007fb88d1921c8 @version="3.5.20.4", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 20>, #<Version::NumericToken 4>]>
release.3.5.20.5 => #<Version:0x00007fb88d1920d8 @version="3.5.20.5", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 20>, #<Version::NumericToken 5>]>
release.3.5.21 => #<Version:0x00007fb88d191e80 @version="3.5.21", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 21>]>
release.3.5.22 => #<Version:0x00007fb88d191d68 @version="3.5.22", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 22>]>
release.3.5.23 => #<Version:0x00007fb88d191c78 @version="3.5.23", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 23>]>
release.3.5.24 => #<Version:0x00007fb88d191b88 @version="3.5.24", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 24>]>
release.3.5.25 => #<Version:0x00007fb88d191a48 @version="3.5.25", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 25>]>
release.3.5.25.1 => #<Version:0x00007fb88d191958 @version="3.5.25.1", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 25>, #<Version::NumericToken 1>]>
release.3.5.25.2 => #<Version:0x00007fb88d191818 @version="3.5.25.2", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 25>, #<Version::NumericToken 2>]>
release.3.5.25.3 => #<Version:0x00007fb88d191728 @version="3.5.25.3", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 25>, #<Version::NumericToken 3>]>
release.3.5.25.4 => #<Version:0x00007fb88d1915e8 @version="3.5.25.4", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 25>, #<Version::NumericToken 4>]>
release.3.5.26 => #<Version:0x00007fb88d1914d0 @version="3.5.26", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 26>]>
release.3.5.27 => #<Version:0x00007fb88d1913b8 @version="3.5.27", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 27>]>
release.3.5.27.1 => #<Version:0x00007fb88d191278 @version="3.5.27.1", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 27>, #<Version::NumericToken 1>]>
release.3.5.3 => #<Version:0x00007fb88d191138 @version="3.5.3", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 3>]>
release.3.5.4 => #<Version:0x00007fb88d190ff8 @version="3.5.4", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 4>]>
release.3.5.5 => #<Version:0x00007fb88d190eb8 @version="3.5.5", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 5>]>
release.3.5.6 => #<Version:0x00007fb88d190d00 @version="3.5.6", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 6>]>
release.3.5.7 => #<Version:0x00007fb88d190b98 @version="3.5.7", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 7>]>
release.3.5.8 => #<Version:0x00007fb88d190aa8 @version="3.5.8", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 8>]>
release.3.5.9 => #<Version:0x00007fb88d190968 @version="3.5.9", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 9>]>
upstream/3.5.1 => #<Version:0x00007fb88d190800 @version="3.5.1", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 1>]>
upstream/3.5.10 => #<Version:0x00007fb88d1906e8 @version="3.5.10", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 10>]>
upstream/3.5.11 => #<Version:0x00007fb88d1904e0 @version="3.5.11", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 11>]>
upstream/3.5.12 => #<Version:0x00007fb88d1903f0 @version="3.5.12", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 12>]>
upstream/3.5.13 => #<Version:0x00007fb88d1902b0 @version="3.5.13", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 13>]>
upstream/3.5.13.pre14 => #<Version:0x00007fb88d1901e8 @version="3.5.13.pre14", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 13>, #<Version::PreToken "pre14">]>
upstream/3.5.14 => #<Version:0x00007fb88d1900f8 @version="3.5.14", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 14>]>
upstream/3.5.15 => #<Version:0x00007fb88d19aaa8 @version="3.5.15", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 15>]>
upstream/3.5.16 => #<Version:0x00007fb88d19bea8 @version="3.5.16", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 16>]>
upstream/3.5.17 => #<Version:0x00007fb88d19bd90 @version="3.5.17", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 17>]>
upstream/3.5.18 => #<Version:0x00007fb88d19bc78 @version="3.5.18", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 18>]>
upstream/3.5.19 => #<Version:0x00007fb88d19bb38 @version="3.5.19", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 19>]>
upstream/3.5.2 => #<Version:0x00007fb88d19b638 @version="3.5.2", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 2>]>
upstream/3.5.20 => #<Version:0x00007fb88d19b3e0 @version="3.5.20", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 20>]>
upstream/3.5.20.2 => #<Version:0x00007fb88d19b188 @version="3.5.20.2", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 20>, #<Version::NumericToken 2>]>
upstream/3.5.20.4 => #<Version:0x00007fb88d19ab98 @version="3.5.20.4", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 20>, #<Version::NumericToken 4>]>
upstream/3.5.20.5 => #<Version:0x00007fb88d19aad0 @version="3.5.20.5", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 20>, #<Version::NumericToken 5>]>
upstream/3.5.21 => #<Version:0x00007fb88d19a9e0 @version="3.5.21", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 21>]>
upstream/3.5.22 => #<Version:0x00007fb88d19a918 @version="3.5.22", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 22>]>
upstream/3.5.23 => #<Version:0x00007fb88d19a850 @version="3.5.23", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 23>]>
upstream/3.5.24 => #<Version:0x00007fb88d19a788 @version="3.5.24", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 24>]>
upstream/3.5.25 => #<Version:0x00007fb88d19a698 @version="3.5.25", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 25>]>
upstream/3.5.25.1 => #<Version:0x00007fb88d19a5a8 @version="3.5.25.1", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 25>, #<Version::NumericToken 1>]>
upstream/3.5.25.2 => #<Version:0x00007fb88d19a490 @version="3.5.25.2", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 25>, #<Version::NumericToken 2>]>
upstream/3.5.25.3 => #<Version:0x00007fb88d19a350 @version="3.5.25.3", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 25>, #<Version::NumericToken 3>]>
upstream/3.5.3 => #<Version:0x00007fb88d19a260 @version="3.5.3", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 3>]>
upstream/3.5.4 => #<Version:0x00007fb88d19a170 @version="3.5.4", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 4>]>
upstream/3.5.5 => #<Version:0x00007fb88d19a058 @version="3.5.5", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 5>]>
upstream/3.5.6 => #<Version:0x00007fb88d199f18 @version="3.5.6", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 6>]>
upstream/3.5.7 => #<Version:0x00007fb88d199e28 @version="3.5.7", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 7>]>
upstream/3.5.8 => #<Version:0x00007fb88d199d38 @version="3.5.8", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 8>]>
upstream/3.5.9 => #<Version:0x00007fb88d199c48 @version="3.5.9", @tokens=[#<Version::NumericToken 3>, #<Version::NumericToken 5>, #<Version::NumericToken 9>]>
djvulibre (guessed) : 3.5.27 ==> 18n-mergepoint
```

</details>

# after the change

```
$ brew livecheck djvulibre
djvulibre : 3.5.27 ==> 3.5.27.1
```

### CODE REVIEW

**Comment by chenrui333** [comment]
Note, The sourceforge one is still not with `3.5.27.1`

File: Livecheckables/djvulibre.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Djvulibre
+  livecheck :url   => "https://git.code.sf.net/p/djvu/djvulibre-git.git",
+            :regex => /^release.(\d+(?:\.\d+)+)$/
```
Comment (samford) [inline Livecheckables/djvulibre.rb]:
```suggestion
            :regex => /^release.?(\d+(?:\.\d+)+)$/
```

Note that `.` in regexes is a wildcard and means "match anything". To match a period, you have to escape it like `\.`. However, in this case, there's always a chance that the format could change from `release.3.5.27` to something like `release-3.5.27`, `release_3.5.27`, or maybe even `release3.5.27`, so I'm okay with using a single wildcard here. We avoid wildcards that match more than one character (e.g., `.*`, `.+`, `.*?`, `.+?`).

I've simply added a `?` here to make it optional, to account for any future releases with typos.

Comment (chenrui333) [reply Livecheckables/djvulibre.rb]: Works for me. Honestly, I dont think we need to be too accommodate with the typo change though.

Comment (samford) [reply Livecheckables/djvulibre.rb]: You may be surprised how often things like this happen and break livecheckables. I'm saying this having seen it firsthand in livecheckables several times now. In this case, we lose nothing by being ever-so-slightly more permissive but we gain a bit by being a little less brittle.

**Pull request merged**